src/index.html

```diff
@@ -7,6 +7,417 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+      href="#"
+      class="header__logo"
+      >
+        <img
+        src="./images/logo.png"
+        alt="logo__MOYO"
+        >
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="logo__MOYO"
        >
```

Add two more spaces for attributes if they are in new lines